src/pages/index.js

```diff
@@ -21,91 +34,155 @@ popUpImageModal.setEventListeners();
 function handleCardClick(name, link) {
   popUpImageModal.openModal(name, link);
 }
-
+//Popup for deleting cards from API along with handler
+const dltPopup = new PopupWithForm(constants.selectors.deleteSelector);
```

Comment:
`dltPopup`

Please, use word `delete` or `confirm` here to make it clear what popup it is


`confirmPopup` or `deletePopup` or `confirmationPopup`